.github/workflows/linting.yml

```
@@ -0,0 +1,27 @@
+name: Lint code
+on:
+  push:
+  pull_request:
+    branches:
+      - main
+    types: [opened, reopened]
+jobs:
+  run-linters:
+    name: Run linters
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out Git repository
+        uses: actions/checkout@v3
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: 20.x
+
+      - name: Install Node.js dependencies
+        run: npm i node @latest
```

Sweet, the addition of `push:` worked as intended, I believe this should be simply the proposed change (or `install` can be shorted to `i`) 
Ref latest [commit that has a failed lint run](https://github.com/Calinator444/newwwie.com/actions/runs/10121760085/job/27993074855)

```suggestion
        run: npm install
```

Otherwise I believe we're 99% the way to consistent code lints :heart_eyes:

---

Before merge, please resolve the above - otherwise the actions in the main branch will be blocked until resolve